tools/clearPersonalDeployment.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# This script truncates all tables in a personal deployment. It is coded to never be able to run against dev.
+# Pass the '-n <namespace>' option to choose which deployment to clear out.
+
+declare NAMESPACE=""
+declare dev_regex="^[dev]"
+declare -a TABLES_TO_TRUNCATE=(databasechangeloglock
+databasechangelog
+dataset
+dataset_column
+snapshot_table
+snapshot
+asset_column
+asset_relationship
+dataset_table
+snapshot_column
+snapshot_source
+snapshot_map_table
+dataset_relationship
+asset_specification
+snapshot_map_column
+project_resource
+billing_profile
+bucket_resource
+load
+load_file
+snapshot_relationship
+dataset_bucket
+storage_resource)
+
+function finish {
+  unset PGPASSWORD
+}
+trap finish EXIT
+
+function stderr {
+    >&2 echo "$@"
+}
+
+function proxy_db_return_pid {
+  local pod=$(kubectl get pod --namespace ${NAMESPACE} | grep sqlproxy | awk '{print $1}')
+  stderr "Connecting to pod ${pod}"
+  kubectl port-forward --namespace tl ${pod} 5433:5432 > /dev/null &
```

Comment:
"tl" needs to be switched to namespace var

Reply:
it looks like you don't have `psql` installed?